source/CCustomOpcodeSystem.cpp

```diff
@@ -2472,10 +2474,21 @@ namespace CLEO
 	//0AC8=2,%2d% = allocate_memory_size %1d%
 	OpcodeResult __stdcall opcode_0AC8(CRunningScript *thread)
 	{
-		DWORD size;
-		*thread >> size;
-		void *mem = malloc(size);
-		if (mem) GetInstance().OpcodeSystem.m_pAllocations.insert(mem);
+		DWORD size; *thread >> size;
+
+		void* mem = malloc(size);

```

Comment:
should we switch over to `calloc` here? I'm not sure how breaking this change might be, but I couldn't imagine any script specifically relying on uninitialized memory

Reply:
Reasonable. Updated.